ERCS/erc-7540.md

```diff
@@ -65,9 +65,9 @@ After submission, Requests go through Pending, Claimable, and Claimed stages. An
 
 | **State**   | **User**                         | **Vault** |
 |-------------|---------------------------------|-----------|
-| Pending     | `requestDeposit(assets, receiver, owner, data)` | `asset.transferFrom(owner, vault, assets)`; `pendingDepositRequest[receiver] += assets` |
-| Claimable   |                                 | *Internal Request fulfillment*:  `pendingDepositRequest[owner] -= assets`; `claimableDepositRequest[owner] += assets` |
-| Claimed     | `deposit(assets, receiver)`      | `claimableDepositRequest[owner] -= assets`; `vault.balanceOf[receiver] += shares` |
+| Pending     | `requestDeposit(assets, requestOwner, assetOwner, data)` | `asset.transferFrom(assetOwner, vault, assets)`; `pendingDepositRequest[assetOwner] += assets` |
```

Comment:
verify this vs. #L587 -- should it the pending step be `pendingDepositRequest[assetOwner] += assets` here?

Reply:
Should be `pendingDepositRequest[requestOwner] += assets` here